defold-spine/src/comp_spine_model.cpp

```diff
@@ -299,10 +299,10 @@ namespace dmSpine
 
     static void ClearCompletionCallback(SpineAnimationTrack* track)
     {
-        if (track->m_AnimationInstance && track->m_AnimationCallbackRef)
+        if (track->m_AnimationInstance && track->m_CallbackInfo)
```

Comment:
Not 100% sure why we check `m_AnimationInstance` here. But decided to keep iit

Reply:
I think the old code was a bit mixed up too. I think it's better now when everything is contained in the callback struct.

We should be able to remove m_AnimationInstance now.